tool/src/main/java/org/datacommons/tool/Tool.java

```diff
@@ -0,0 +1,231 @@
+package org.datacommons.tool;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.datacommons.proto.Debug;
+import org.datacommons.proto.Mcf;
+import org.datacommons.util.McfParser;
+import org.datacommons.util.McfUtil;
+import org.datacommons.util.TmcfCsvParser;
+import picocli.CommandLine;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+// TODO: Add e2e tests once Debug.Log is fully plumbed in.
+@CommandLine.Command(
+    name = "dc-import",
+    mixinStandardHelpOptions = true,
+    version = "dc-import 0.1",
+    description = "Tool for use in developing datasets for Data Commons.",
+    subcommands = {Lint.class, GenMcf.class})
+class Tool {
+  @CommandLine.Option(
+      names = {"-o", "--outputDir"},
+      description = "Directory to write output files. Default is current working directory.",
+      scope = CommandLine.ScopeType.INHERIT)
+  public File outputDir;
+
+  public static void main(String... args) {
+    System.exit(new CommandLine(new Tool()).execute(args));
+  }
+
+  public static void HandleNodeFormat(
```

Comment:
can we rename Tool to something that more clearly denotes that this is where `main` is? Main or Runner?

handleNodeFormat and handleTabFormat should also go in a separate class so that Lint and GenMcf can go in separate files. We should try to maintain one class per file. should we also give them more specific names? parseNodeFormat or parseTableFormat?

Reply:
1. Renamed Tool to Main
2. Split the two subcommands into their own file
3. Moved the common helpers into a Processor container class.

RE: giving helper funcs specific names, leaving it as a somewhat general name (processFoo), because I was thinking that these functions will drive the work (like syntax check, stat check, resolution, etc.) for every node/sub-graph parsed in a streaming way (so people can pass in a very large input and we can report partial/incremental results).

Reply:
And, wow, IDE makes refactoring a breeze :)

Reply:
I will merge this for ease of follow on changes, and incorporate comments on the above changes in a follow-on PR.